client/src/features/Communication/components/NewThread.tsx

```diff
@@ -10,8 +9,35 @@ import {
   Button,
 } from "@fluentui/react-components";
 import {TextField} from "@fluentui/react";
+import {useState} from "react";
 
 export const FluidActions = () => {
+  const [threadName, setThreadName] = useState("");
+  const onChatButton = async () => {
+    try{
+      console.log("Button works");
+      const response = await fetch('http://localhost:5103/communication/createThread', {
```

Comment:
Can we move this API call to singleton class in API folder? Also, I see that you are using the full path which uses the local web server network and port name for the flask app. Since we are using an API Gateway, you should be able to use /communication/createThread, and it will reroute the request to the appropriate service

Reply:
Yeah I'll move it to the API folder, I have been keeping the full path for now for testing reasons but once we are ready for production I will remove them.